cloud_edition/flexibility_mode/docs/environments_access.rst

```diff
@@ -102,25 +107,30 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
 .. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit
 
 
+Data Transfer Between Instances
+----------------------------------
```

Comment:
If there is too many "----" then the size of the title, the title will not be generated correctly